### ISSUE TITLE
How to properly compile the .jar using gradlew

### ISSUE THREAD

**Issue by FreakySt0rm** [SwipeX/PokeMate#48]
Hi, could you supply instructions for compiling the .jar from the updated source? I keep getting issues when I try to build the project using gradlew :\

**Comment by jeffsnow88** [comment]
Hi, I just download the file in .zip format, extracted and then imported in intelijj IDEA as recommended by SwipeX.

However, i'm still having problems in login part. Hope that can help you.

**Comment by FreakySt0rm** [comment]
Thanks for that Jeff. Just downloading intelijj IDEA Ultimate. I'll let you know how it goes ;)

**Comment by kyleshrader** [comment]
![image](https://cloud.githubusercontent.com/assets/5632571/17117805/c4b19d3a-5272-11e6-9017-a16b85265160.png)

In intellij it should just work with this...

**Comment by kyleshrader** [comment]
Make sure you have the plugin installed for gradle. I am using community version

**Comment by jeffsnow88** [comment]
Np man. Let me know if  you can pass login part. Im stuck there.

**Comment by TaintedSynchro** [comment]
@kyleshrader How do you install the gradle plugin?

**Comment by jc030201** [comment]
@kyleshrader how do you install the gradle plugin?

**Comment by eBunny6** [comment]
[http://prnt.sc/bxgv7c](url)

Im lost. what am i doing wrong.

**Comment by eBunny6** [comment]
![untitled](https://cloud.githubusercontent.com/assets/20648904/17119327/6f12db0c-52c5-11e6-89cf-7733e9ed473b.jpg)

Im lost, and stuck. What am i doing wrong?

**Comment by SwipeX** [comment]
@eBunny6 right click on build.gradle and select run
Then right click on PokeMate.java and select run

**Comment by kyleshrader** [comment]
![image](https://cloud.githubusercontent.com/assets/5632571/17128382/26b5853e-52c1-11e6-9431-4f8a7bc38155.png)
Clicking this button will show the console output... you can get the auth code and put it in the google.com/devices site... and from there you're good

**Comment by eBunny6** [comment]
I got it past the code for google devices, now im stuck with some error from pokemon servers. It similar to the one error from the closed issue of TheJeffSnow88. Im posting from mobile, and i cant link anything. As soon as i get home from work ill post more details.

**Comment by eBunny6** [comment]
Ok, i resolved all the issues i had, its running smoothly. Still, how do i get it to show the UI? Its all happening in IntelliJ, text only. Run PokeMateUI?

**Comment by rawkus123** [comment]
yeah i have no idea how to run the GUI either... console runs sweet, and i can load the UI in IDEA aswell but nothing populates.

**Comment by BurakDev** [comment]
If you want build a standalone jar with all dependencies use this command : gradle fatJar (or ./gradlew fatJar if you don't have gradle)

**Comment by rawkus123** [comment]
ok but still doesnt open up a GUI...

**Comment by BurakDev** [comment]
As @SwipeX said, you need run it from command line if you use Google Auth.

How are you running it ?

**Comment by rawkus123** [comment]
ill list the steps ive used so far...
Download source / zip... extract to x:\PokeMate....
Config.properties changed
Download PokeMate.jar from releases tab place in x:\PokeMate
Run from cmdline: java -jar pokemate.jar
starts command line bot.....

Install IDEA and import project....
Run from Application-Run in right hand pane view of grandle
Runs command line bot inside IDEA....

Run gradlew.bat from CMD ... Gradle completes (checked via gradlew from prompt)
use cmd: gradlew fatjar ... build is successful
use cmd: gradlew run
starts cmd line bot
all these attempts are fresh attempts aswell new directory etc...
Forgot to mention the bots run fine in cmd though... just would like a GUI lol

**Comment by rawkus123** [comment]
moved my issue to topic [#106](https://github.com/SwipeX/PokeMate/issues/106)

**Comment by eBunny6** [comment]
Everything was working smooth, until couple of hrs ago. Now when i run it, it just says i need to change config file. No matter what, it just displays single line of text, change config.properties in order to run this program.

**Comment by MihaelBercic** [comment]
For the UI to show up, change the property show=false to show=true in config file.
